HIP-0013.md

```diff
@@ -0,0 +1,130 @@
+
+The general syntax for profile data is:
+
+**TXT** `profile <attribute name>=<attribute value>`
+
+There are currently five supported attributes: **[email](#email)**, **[phone](#phone)**, **[avatar](#avatar)** and **[service](#service)**. All attributes are optional but at least one attribute needs to be present, for a Domain to count as a profile.
+
+Although each `<attribute name>=<attribute value>` pair can be provided as individual records, please consider using the **[recommended implementation](#alt)**.
+
+### <a id="email"></a> (1) Profile Attribute: Email
+
+The email attribute is optional. It enables a domain owner to provide an email for the domain. The expected value is an email address.
+
+```
+TXT profile email=<email-address>
+```
+
+#### Valid Email Examples
+
+```
+TXT profile email=hello@niami.io
+TXT profile email=hello@niami
+```
+
+### <a id="phone"></a> (2) Profile Attribute: Phone
+
+The phone attribute is optional. It enables a domain owner to provide a phone number for the domain. The expected value is a phone number as an E.164 string (a number up to fifteen digits in length starting with a '+'). Regex: `^\+[1-9]\d{1,14}$`
+
+```
+TXT profile phone=<phone-number>
+```
+
+#### Valid Phone Examples
+
+```
+TXT profile phone=+15368886253
+TXT profile phone=+47538886253
+```
+
+### <a id="avatar"></a> (3) Profile Attribute: Avatar
+
+The avatar attribute is optional. It enables a domain owner to provide an avatar for the domain. The expected value is a fully qualified URL that must start with http:// or https:// that points to an image file.
+
+Allowed image formats are: APNG, AVIF, GIF, JPEG, PNG, SVG, WebP according to [Common image file types](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types).
+
+```
+TXT profile avatar=<URL>
+```
+
+#### Valid Avatar Examples
+
+```
+TXT profile avatar=https://mywebsite.com/avatar.png
+TXT profile avatar=https://siasky.net/MABn4I5M6yPT0aCjIMU2OjlTWmKysaqoCeP0gSJw5moVrg
+```
+
+### <a id="service"></a> (4) Profile Attribute: Service
+
+The service attribute is optional. It enables a domain owner to provide a username to a service. The expected value is a service key followed by a username.
+
+```
+TXT profile service=<service-key username>
+```
+
+The service key is made up of a reverse dot notation for a namespace the service owns, ex: com.twitter, com.namebase, io.github. Service keys must contain at least one dot. So if a service is hosted on a Handshake TLD, the service key has to end with a dot.
+
+#### Valid Service Examples
+
+```
+TXT profile service=com.twitter niami
+TXT profile service=com.github 0xstefan
+TXT profile service=io.keybase pinheadmz
+TXT profile service=hnschat. skmo
+```
+
+## <a id="alt"></a> Recommended Implementation
+
+To save blockchain / urkel tree space it's encouraged to use only one TXT record for the profile. A semicolon is then used to seperate each `<attribute name>=<attribute value>` pair.
```

Comment:
Is semicolon (the delimiter) prohibited as values? If no, are they supposed to be escaped? Maybe add a note somewhere.
Curious how your fourhands implementation handles it.